scripts/sanity_chk/sanitylib.py

```diff
@@ -2313,7 +2313,7 @@ def summary(self, unrecognized_sections):
                 (100 * len(self.selected_platforms) / len(self.platforms))
             ))
 
-    def save_reports(self, name, report_dir, no_update, release, only_failed):
+    def save_reports(self, name, suffix, report_dir, no_update, release, only_failed):
```

Comment:
can we move new add suffix to the last parameter and add a default vaule for it suffix=None, such as other scripts calling this save_reports will be not impacted by this change